src/connectedmachine/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.5.1'
```

Comment:
In fact, we only recommend upgrading to the major version when there is a breaking change. If this PR does not include the breaking change, could we only upgrade to the secondary version?

Reply:
Updated version number to 0.6.0